src/overlays/actors/ovl_En_Arrow/z_en_arrow.c

```diff
@@ -251,7 +251,7 @@ void func_8088A894(EnArrow* this, PlayState* play) {
         return;
     }
 
-    temp_f0 = Math3D_LengthSquared(&sp68);
+    temp_f0 = Math3D_Vec3fMagnitudeSq(&sp68);
     if (temp_f0 < 1.0f) {
```

Comment:
I know it is just 1 but should we still do `SQ(1.0f)`? (assuming it still matches of course)

Reply:
Elliptic and I had a similar conversation in one of Elliptics PR a long time ago, but I can't seem to find it. Basically we decided that `SQ(1.0f)` wasn't really needed, and you see formulas like `1 - x^2` all the time, but don't really see `1^2 - x^2`